pkg/apis/autoscaling/annotation.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package autoscaling
+
+import "strings"
+
+func Inherit(to, from map[string]string) map[string]string {
```

Comment:
Added GoDoc for both.

Did you mean `InheritAutoscaling` in your suggestion above?  I named it only `Inherit` because it's part of the `autoscaling` package.

 I split the annotation inheritance into a separate PR.  Continue review of this file in https://github.com/knative/serving/pull/2369.